core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -462,6 +462,15 @@ private[deploy] class Worker(
           // when cleaning up
           val appIdFromDir = dir.getName
           val isAppStillRunning = appIds.contains(appIdFromDir)
+
+          // Remove some registeredExecutors information of DB in external shuffle service when
+          // #spark.shuffle.service.db.enabled=true, the one which comes to mind is, what happens
+          // if an application is stopped while the external shuffle service is down?
+          // So then it'll leave an entry in the DB and the entry should be removed.
+          if (conf.get(config.SHUFFLE_SERVICE_DB_ENABLED) &&
+            conf.get(config.SHUFFLE_SERVICE_ENABLED) && !isAppStillRunning) {
```

Comment:
nit: double indent this line, the continuation of the condition

Reply:
Hi,@squito ,i have fixed it，thanks a lot!